clustermesh-apiserver/clustermesh/root.go

```diff
@@ -358,7 +401,13 @@ func startServer(
 		"cluster-id":   cinfo.ID,
 	}).Info("Starting clustermesh-apiserver...")
 
-	synced.SyncCRDs(startCtx, clientset, synced.ClusterMeshAPIServerResourceNames(), &synced.Resources{}, &synced.APIGroups{})
+	syncErr, err := crdSyncPromise.Await(startCtx)
+	if err != nil || syncErr != nil {
+		if err == nil {
+			err = syncErr
+		}
+		log.WithError(err).Fatal("Wait for CRD resources failed")
+	}
```

Comment:
This logic could be dropped altogether, as the logic below only fetches events using `Resource[T]`.

Reply:
Good to know! Done.